README.md

```diff
@@ -56,7 +56,7 @@ The add-in provides the ability to create geodesic features such as lines, circl
 * Running the add-in
 	* To run from a stand-alone deployment
 		* ArcMap
-			* Install the add-in from the application folder by double clicking the **ArcMapAddinDistanceAndDirection.esriAddIn** file.
+			* Install the add-in from the latest Release on Github (https://github.com/Esri/distance-direction-addin-dotnet/releases)
```

Comment:
I was just going to do whatever you all were doing for the Pro Symbol Editor so I was just curious - I like the idea of attaching the pre-built binary to the release so I think I will go with that until someone tells me differently - thanks